tests/Tests.FeatureManagement/Features.cs

```diff
@@ -5,6 +5,7 @@ namespace Tests.FeatureManagement
 {
     enum Features
     {
+        Beta,
```

Comment:
Everything looks good. Just one minor suggestion - change the flag name to something like `TargetingTestFeature`. It will make it easier to read and maintain the test code.

Reply:
Good suggestion, updated.